cime/config/e3sm/testmods_dirs/allactive/v1cmip6/shell_commands

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 ./xmlchange --append CAM_CONFIG_OPTS='-cosp'
 ./xmlchange --id BUDGETS --val TRUE
-./xmlchange PIO_TYPENAME=netcdf
```

Comment:
> This update to maint-1.0 will be brought forward to master later.

This update didn't make it to master yet, but current master can run `SMS.ne30_oECv3_ICG.A_WCYCL1850S_CMIP6.anvil_intel` with `PIO_TYPENAME=pnetcdf` without any errors if `PIO_TYPENAME=2` (and errors out with the current default of `PIO_TYPENAME=1`).

Jayesh (@jayeshkrishna), just checking: will the internal switch to read netcdf4 files continue?
Benchmarks show:
a. netcdf+scorpio+master is slower than pnetcdf+scorpio_classic+maint-1.0, and
b. pnetcdf+scorpio+master is faster than pnetcdf+scorpio_classic+maint-1.0.

If support for netcdf4 in scorpio will go away, then we probably can't have `b.` until the netcdf4-to-netcdf3 update on master.

Reply:
So master works with PIO_VERSION=2 because it does a better job of quietly switching to netcdf4 when a netcdf4 file is opened?

Reply:
Yes. There is an info note in e3sm.log:
```
   0: PIO: WARNING: Opening file (/home/ccsm-data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a4_elev_1850-2014_c170525.nc) with iotype=1 failed. Switching iotype to PIO_IOTYPE_NETCDF
```
And file type is
```
$ ncdump -k /home/ccsm-data/inputdata/atm/cam/chem/trop_mozart_aero/emis/DECK_ne30/cmip6_mam4_num_a2_elev_1850-2014_c170525.nc
netCDF-4 classic model
```